buildlib/tools/builds.sh

```diff
@@ -461,5 +461,5 @@ do
 	[ -d "${ucx_build_dir}" ] && rm -rf ${ucx_build_dir}/*
 
 	# run the test
-	${test_name}
+	${test_name} || { echo "Test failed: ${test_name}"; exit 1; }
```

Comment:
if we exit, we cannot get a list of failing tests at once, which could be painful?

Reply:
I can remove the "exit 1;" part

Reply:
the test would already exit after first failure, to not consume CI resources
https://github.com/openucx/ucx/blob/master/buildlib/tools/builds.sh#L1

Reply:
I see

Reply:
removed the "exit 1;" part @tvegas1 @yosefe

Reply:
pls keep the exit

Reply:
done @yosefe